src/gui/remote/DatabaseSettingsWidgetRemote.cpp

```diff
@@ -46,8 +46,10 @@ DatabaseSettingsWidgetRemote::DatabaseSettingsWidgetRemote(QWidget* parent)
     connect(m_ui->nameLineEdit, &QLineEdit::textChanged, setModified);
     connect(m_ui->downloadCommand, &QLineEdit::textChanged, setModified);
     connect(m_ui->inputForDownload, &QPlainTextEdit::textChanged, setModified);
+    connect(m_ui->downloadTimeout, QOverload<int>::of(&QSpinBox::valueChanged), setModified);
```

Comment:
Since you don't use the value from this signal you could just use `textChanged` to avoid the awkward QOverload

Reply:
I'm not sure why but the Ubuntu build does not agree
```
/opt/buildagent/work/c401303cba1b4098/src/gui/remote/DatabaseSettingsWidgetRemote.cpp: In constructor ‘DatabaseSettingsWidgetRemote::DatabaseSettingsWidgetRemote(QWidget*)’:
/opt/buildagent/work/c401303cba1b4098/src/gui/remote/DatabaseSettingsWidgetRemote.cpp:49:50: error: ‘textChanged’ is not a member of ‘QSpinBox’
   49 |     connect(m_ui->downloadTimeoutSec, &QSpinBox::textChanged, setModified);
      |                                                  ^~~~~~~~~~~
/opt/buildagent/work/c401303cba1b4098/src/gui/remote/DatabaseSettingsWidgetRemote.cpp:52:48: error: ‘textChanged’ is not a member of ‘QSpinBox’
   52 |     connect(m_ui->uploadTimeoutSec, &QSpinBox::textChanged, setModified);
      |                                                ^~~~~~~~~~~
make[2]: *** [src/CMakeFiles/keepassxc_gui.dir/build.make:1254: src/CMakeFiles/keepassxc_gui.dir/gui/remote/DatabaseSettingsWidgetRemote.cpp.o] Error 1
make[2]: *** Waiting for unfinished jobs....
make[1]: *** [CMakeFiles/Makefile2:677: src/CMakeFiles/keepassxc_gui.dir/all] Error 2
make: *** [Makefile:141: all] Error 2
Process exited with code 2
```

Reply:
Ugh it was first introduced in qt 5.14. Sorry for the bad recommendation, have to revert back to valueChanged